libraries/AP_UAVCAN/dsdl/ardupilot/equipment/gnss/20002.Inject.uavcan

```diff
@@ -0,0 +1,4 @@
+#
+# GNSS Inject, used by autopilots to forward GPS_INJECT mavlink msgs
+# to GNSS nodes. Normally sent as broadcast
+uint8[<=110] data
```

Comment:
You should use the newer "GPS_RTCM_DATA", it contains upto 4*180 bytes, instead of the older "GPS_INJECT_DATA" that only supports 110 bytes.

Reply:
yes, good point, thanks

Reply:
I've fixed it to us the same flags conventions as GPS_RTCM_DATA